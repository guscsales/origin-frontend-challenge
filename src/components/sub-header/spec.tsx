import * as React from 'react';
import { create } from 'react-test-renderer';
import theme from '../../configs/styles/themes';
import { ThemeProvider } from 'styled-components';
import SubHeader from '.';

describe('SubHeader', () => {
  describe('#render', () => {
    it('should match a snapshot', () => {
      const wrapper = create(
        <ThemeProvider theme={theme}>
          <SubHeader />
        </ThemeProvider>
      ).toJSON();

      expect(wrapper).toMatchSnapshot();
    });
  });
});
