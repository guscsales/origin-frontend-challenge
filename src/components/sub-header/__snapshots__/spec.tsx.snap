// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader #render should match a snapshot 1`] = `
.c0 {
  margin: 3rem 0;
}

.c1 {
  font-family: Work Sans;
  font-size: 2.1rem;
  font-weight: 400;
  margin: 0;
  text-align: center;
  color: #1b31a8;
}

.c1 strong {
  font-weight: 600;
}

.c1 span {
  display: block;
  font-size: 1.5rem;
  font-weight: 400;
  margin-top: 0.4rem;
  color: #657786;
}

@media (min-width:1024px) {
  .c0 {
    margin: 4.5rem 0;
  }
}

@media (min-width:1024px) {

}

<div
  className="c0"
>
  <h2
    className="c1"
  >
    Let's plan your 
    <strong>
      saving goal.
    </strong>
  </h2>
</div>
`;
